AndroidSDK/src/com/leanplum/Leanplum.java

```diff
@@ -628,6 +629,9 @@ private static void startHelper(
     Date now = new Date();
     int timezoneOffsetSeconds = localTimeZone.getOffset(now.getTime()) / 1000;
 
+    String registrationId = SharedPreferencesUtil.getString(context,
```

Comment:
looks like in iOS: we also include appId etc.: https://github.com/Leanplum/Leanplum-iOS-SDK/pull/39/files#diff-37d7318a53e27e0c036850d334aace01R427
do we need this for android?